### DISCUSSION TITLE
just a suggestion: ad module

### DISCUSSION THREAD

**Discussion by jackhacksren** [ftde0/yt2009#187] [Ideas]
is it possible to put the ad module into yt2009 or atleast emulate it and even at flash form

it can be enabled by checking emulate_ads  in the flags

**Comment by ftde0** [comment]
haven't messed with the ad module as i won't be adding ads into yt2009. leaving this open however as a reminder

**Comment by SGR21gtb** [comment]
Any update? I really want to try out ads on YT2009 (mobile and web), which i'm making a new issue on...
